### ISSUE TITLE
Does not parse <content:encoded> and <guid> elements (among others)

### ISSUE THREAD

**Issue by p3k** [bobby-brennan/rss-parser#8]
Trying to use rss-parser with Instant Article feed the following RSS results in incomplete data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<rss xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:dc="http://purl.org/dc/elements/1.1/" version="2.0">
  <channel>
    <title>Instant Article Test</title>
    <link>https://localhost:8000</link>
    <description>1, 2, 1, 2… check the mic!</description>
    <language>en</language>
    <pubDate>Fri, 13 May 2016 15:14:05 GMT</pubDate>
    <dc:date>2016-05-13T15:14:05Z</dc:date>
    <dc:language>en</dc:language>
    <item>
      <title>My first Instant Article</title>
      <link>https://localhost:8000</link>
      <description>Lorem ipsum</description>
      <content:encoded>&lt;b&gt;Lorem&lt;/b&gt; ipsum</content:encoded>
      <pubDate>Wed, 04 May 2016 06:53:45 GMT</pubDate>
      <guid>https://localhost:8000</guid>
      <dc:creator>tobi</dc:creator>
      <dc:date>2016-05-04T06:53:45Z</dc:date>
    </item>
  </channel>
</rss>
```

This is the result in JSON; note the missing `guid` and `content:encoded` data:

```json
{
  "feed": {
    "entries": [
      {
        "title": "My first Instant Article",
        "link": "https://localhost:8000",
        "pubDate": "Wed, 04 May 2016 06:53:45 GMT",
        "content": "Lorem ipsum",
        "contentSnippet": "Lorem ipsum"
      }
    ],
    "title": "Instant Article Test",
    "description": "1, 2, 1, 2… check the mic!",
    "link": "https://localhost:8000"
  }
}
```

**Comment by consoles** [comment]
I have the same problem.

**Comment by bobby-brennan** [comment]
I just added the example you gave as a test case, and both guid and content:encoded seem to be working for me.

Can you pull the latest version and run `npm test`?

**Comment by andjosh** [comment]
Just ran `npm test`, with results:

~~~
$ npm test

> rss-parser@2.2.4 test rss-parser
> mocha



  Parser
    1) should parse Reddit
    ✓ should not parse craigslist (RSS 1.0)
    2) should parse atom
    ✓ should parse atom feed
    ✓ should parse reddits new feed
    ✓ should parse with missing fields
    ✓ should parse heise


  5 passing (153ms)
  2 failing

  1) Parser should parse Reddit:

      Uncaught AssertionError: expected { Object (feed) } to deeply equal { Object (feed) }
      + expected - actual

               "pubDate": "Thu, 12 Nov 2015 12:43:37 +0000"
               "title": "Tranquility in the woods [2765x1834] Almora, India Photograph by Mayank Pant"
             }
           ]
      -    "feedUrl": [undefined]
           "link": "https://www.reddit.com/"
           "title": "reddit: the front page of the internet"
         }
       }
      
      at tryToString (fs.js:414:3)
      at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:401:12)

  2) Parser should parse atom:

      Uncaught AssertionError: expected { Object (feed) } to deeply equal { Object (feed) }
      + expected - actual

               "pubDate": "Sat, 23 Jan 2016 16:28:07 +0000"
               "title": "ELI5: Why does a computer sometimes have a hard time starting up, and what exactly does a Start Up Repair do to fix it?"
             }
           ]
      -    "feedUrl": [undefined]
           "link": "https://www.reddit.com/"
           "title": "reddit: the front page of the internet"
         }
       }
      
      at tryToString (fs.js:414:3)
      at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:401:12)



npm ERR! Test failed.  See above for more details.
~~~

**Comment by andjosh** [comment]
It's probably because you didn't release a new version of the package to bower or npm. The latest release is from February, but the fixes you are referring to have happened since then. Bump the version, release, and things may be better.

**Comment by bobby-brennan** [comment]
Thanks, released 2.3.0 on both npm and bower

**Comment by SteffenKohlhagen** [comment]
Hey Bobby,

I have a small question. My taget is „print in console.log" all Content from the node <content:encoded>

This is okay:
console.log("item" + entry.pubDate);
Print: itemMon, 02 Oct 2017 20:24:53 +0000

Question: 
What I have to do for print <content:encoded> in console?
console.log("item" + entry.XXXX);

Do you have a tipp for me?

This is the code:
```
var options = {
            customFields: {
                channel: ['otherTitle', 'extendedDescription'],
                item: ['title', 'description', 'pubDate','link', 'content:encoded'],
            }
        }
        // Artikelparser
        parser.parseURL('myfeed', options, function(err, parsed) {
            console.log(parsed);
    
            parsed.feed.entries.forEach(function(entry) {                                
                var item = [entry.title, entry.description, entry.pubDate, entry.link, entry.content];
		console.log("item" + entry.XXXX);
                console.log("item" + entry.pubDate);
```
The Feed:

```
<item>
	
		<description><![CDATA[Wenn Tante Ri spricht, erzittern die Feinde vor Angst. Pjöngjang, Propaganda und die Prinzipien der freien Presse – wie Westmedien in Nordkorea arbeiten und wie sich das nordkoreanische vom westlichen Journalismusverständnis unterscheidet. Eine Bestandsaufnahme. VORIGER ARTIKEL Van Morrison – Blues für Sieger Artikel veröffentlicht: Donnerstag, 21.09.2017 17:21 Uhr NÄCHSTER ARTIKEL Die Wahlgesänge: Tipps für die [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>Wenn Tante Ri spricht, erzittern die Feinde vor Angst. Pjöngjang, Propaganda und die Prinzipien der freien Presse – wie Westmedien in Nordkorea arbeiten und wie sich das nordkoreanische vom westlichen Journalismusverständnis unterscheidet. Eine Bestandsaufnahme.</p>
<p>VORIGER ARTIKEL<br />
Van Morrison – Blues für Sieger<br />
<p>2012 habe sie die Eröffnung eines Büros in einem Land, mit dem die USA faktisch noch im Krieg liegen, mit Champagner gefeiert, erinnert sich Lee. „Dieser Moment fühlte sich an wie ein Sieg, aber in Wahrheit hat die Schlacht gerade erst begonnen.“</p>
<p>Von Imre Grimm / RND</p>
]]></content:encoded>
	
		</item>
```

**Comment by bobby-brennan** [comment]
It should be available as `item['content:encoded']` - is it not?
